### ISSUE TITLE
ninja build error

### ISSUE THREAD

**Issue by wjhtinger** [manu88/Sofa#44]
/home/wangjh/work/sofa/Sofa/projects/Sofa/libSofa/src/runtime.c: In function ‘RuntimeInit’:
/home/wangjh/work/sofa/Sofa/projects/Sofa/libSofa/src/runtime.c:96:8: error: ‘struct env’ has no member named ‘io_space’
   96 |     env.io_space = init_data->io_space;
      |        ^

What's wrong, any ideas?

**Comment by cagOs** [comment]
yes, I also saw the issue when i build the project，i think these commit does not upload the all project files.

**Comment by ventrue13** [comment]
So have u solved?
